src/components/Badge/styled/badge.ts

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components/native';
+import { BaseProps } from '../../types';
+import {
+    COLOR_GRAY,
```

Comment:
I think that we can use the color `${COLOR_GRAY_TRANSPARENT_2} instead create a new color_gray